types/babel__traverse/index.d.ts

```diff
@@ -90,7 +90,13 @@ export class Scope {
 
     removeData(key: string): void;
 
-    push(opts: any): void;
+    push(opts: {
+        id: object,
```

Comment:
Should be of type string?

Reply:
`id` and `init` get passed into the builder for `VariableDeclarator`, so presumably they take the same types as the builder - `LVal` and `Expression` respectively.